activerecord/lib/active_record/tasks/database_tasks.rb

```diff
@@ -252,6 +251,18 @@ def load_seed
         end
       end
 
+      def self.run_cmd(cmd, args, action)
+        command = args.dup.unshift([cmd, cmd])
```

Comment:
We don't. It only gives the process a specific name ("sometimes" according to docs). Will change it.